### PR TITLE
Navbar/Settings overhaul and many small changes

### PR DESCRIPTION
**Pull request by MeemawHustlin**
The commits summarize all the small things. This addresses PR #282 in addition to tons of minor fixes and updates.

### CODE REVIEW

File: js/cellGenerator.mjs [hunk 1]
```diff
@@ -74,27 +74,19 @@ function generateLinkUrl(cell, format){
             return null;
         }
         else{
-            linkHref = "./map.html?formId="+cell.formId;
+            linkHref = "./map.html?formId=" + cell.formId;
             if((format & CELL_FORMAT_NAMELINK_OPEN_IN_IFRAME) && window.settings.iframeCheck == "on"){
```
Comment (MichaelEbert) [inline js/cellGenerator.mjs:78]: change this iframeCheck to != "off" instead of forcing topbar to false for all maps.

File: js/cellGenerator.mjs [hunk 2]
```diff
@@ -150,11 +142,11 @@ function createLinkElement(cell, linkName, format){
         linky.innerText = capitalClassName + linkName;
     }
 
-    if(format & CELL_FORMAT_NAMELINK_MAPLINK){
-        linky.title = "View on map";
-    }
     if(format & CELL_FORMAT_NAMELINK_SEPARATE_HELP){
         linky.title = "View on UESP"
+    }
+    if(format & CELL_FORMAT_NAMELINK_MAPLINK){
```
Comment (MichaelEbert) [inline js/cellGenerator.mjs:148]: should probably be an "else if" instead of just another if statement.

**Review by MichaelEbert — CHANGES_REQUESTED** [review]
Looks mostly good. Thank you for adding so much descriptions to the options!

File: js/cellGenerator.mjs (hunk 1 above)
Comment (MeemawHustlin) [reply js/cellGenerator.mjs]: I think I did it correctly, should be fixed now.

File: js/cellGenerator.mjs (hunk 2 above)
Comment (MeemawHustlin) [reply js/cellGenerator.mjs]: fixed.

**Review by MichaelEbert — APPROVED** [review]

**Pull request merged**